### PR TITLE
Explore: Explore Log Download doesn't have same time format

### LINKED ISSUE #94486: Logs: downloaded logs use different timezones
When exporting log lines from the explore page, the export button has three formats: json,csv & txt, from that txt file is taking local browser time, where as json and csv are in UTC epoch time stamp. The expectation is that the three methods use the same format.

### PR DESCRIPTION
**Pull request by yincongcyincong**
Fixes #94486

### CODE REVIEW

**Review by matyax — CHANGES_REQUESTED** [review]
Hi @yincongcyincong, thanks a lot for taking a stab at this bug. It was not explicit in the issue, but what we need to do is to remove the timezone change that occurs in https://github.com/grafana/grafana/blob/main/public/app/features/inspector/utils/download.ts#L33 , so that all files are in UTC.

If you are down to fixing this bug, please revert these changes and focus on download.ts

**Comment by yincongcyincong** [comment]
> Hi @yincongcyincong, thanks a lot for taking a stab at this bug. It was not explicit in the issue, but what we need to do is to remove the timezone change that occurs in https://github.com/grafana/grafana/blob/main/public/app/features/inspector/utils/download.ts#L33 , so that all files are in UTC.
> 
> If you are down to fixing this bug, please revert these changes and focus on download.ts

thanks for the review. i will have a try to fix it.

**Review by matyax — APPROVED** [review]
Thank you!

**Comment by matyax** [comment]
@yincongcyincong Can you merge or rebase with the latest main? Thanks!

**Comment by yincongcyincong** [comment]
> @yincongcyincong Can you merge or rebase with the latest main? Thanks!

bro. i have merged the latest main. thanks

File: public/app/features/inspector/utils/download.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export function downloadLogsModelAsTxt(logsModel: Pick<LogsModel, 'meta' | 'rows
   textToDownload = textToDownload + '\n\n';
 
   logsModel.rows.forEach((row) => {
-    const newRow = dateTimeFormat(row.timeEpochMs, { defaultWithMS: true }) + '\t' + row.entry + '\n';
```
Comment (matyax) [inline public/app/features/inspector/utils/download.ts:33]: You probably need to also update download.test.ts https://github.com/grafana/grafana/commit/af3ea54d0e8a5c84e25d6473bc7a2f53b6cdd960#diff-f4d98a123cd5f0f5b9f5ce16bb57b66882cffbbcb9ef6624aed059d836d304bd

**Comment by svennergr** [comment]
FWIW, we just got another bug report for wrongly ordered logs: https://github.com/grafana/grafana/issues/94601

Feel free to fix the behavior in this PR as well, if you want.

Thank you!

**Comment by yincongcyincong** [comment]
> FWIW, we just got another bug report for wrongly ordered logs: #94601
> 
> Feel free to fix the behavior in this PR as well, if you want.
> 
> Thank you!

thanks give me this info. i will have a try.

File: public/app/features/inspector/utils/download.ts (hunk 1 above)
Comment (yincongcyincong) [reply public/app/features/inspector/utils/download.ts:33]: thanks. bro. i will modified it.

**Comment by yincongcyincong** [comment]
@matyax @svennergr could you guys please take a look newest commit code. i modified csv order issue.

**Comment by yincongcyincong** [comment]
> @yincongcyincong Thanks a lot for giving a try to #94601, but it requires support to both dataplane and non-dataplane fields, which opens a whole can of worms unrelated with this PR.
> 
> Let's just keep the changes in download.ts to fix #94486 and address #94601 somewhere else.

thanks for your review. i will handle it.

**Comment by yincongcyincong** [comment]
@matyax bro, i deleted the change about csv order. please take a look.

**Review by matyax — APPROVED** [review]

**Comment by matyax** [comment]
@yincongcyincong Thank you so much for your patience and willingness to iterate on this issue with us. Very much appreciated.

**Pull request merged**